### DISCUSSION TITLE
Duplicate plugin not detecting duplicates?

### DISCUSSION THREAD

**Discussion by augustross3** [beetbox/beets#5247] [Q&A] [answered]
Hello. Fresh database and I import the following folder as a test

ls ~/music4/Dj.Mc/
'Dj.Mc - 5,4,3,2,1'
'Dj.Mc - Fall Back'
'Dj.Mc - Relentless EP'
'Dj.Mc - Relentless EP2'
'Dj.Mc - The Lowend'

I duplicate the album "Relentless" to test duplication function. All metadata and everything with the files is unchanged thus they should be duplicates.

I import with
beet import -A ~/music4/Dj.Mc/ and get

/music4/Dj.Mc/Dj.Mc - 5,4,3,2,1
/music4/Dj.Mc/Dj.Mc - Fall Back
/music4/Dj.Mc/Dj.Mc - Relentless EP
/music4/Dj.Mc/Dj.Mc - Relentless EP2
/music4/Dj.Mc/Dj.Mc - The Lowend

along with beet -ls get clear duplicates example:

Dj.Mc - Relentless EP - 5,4,3,2,1
Dj.Mc - Relentless EP - 5,4,3,2,1
Dj.Mc - Relentless EP - House Goin Viral
Dj.Mc - Relentless EP - House Goin Viral

So I enable and run duplicates plugin and do

beet duplicates

which returns nothing. Am I doing something wrong here? Note in this particular test there was nothing important with MusicBrainz as none of these albums/eps/singles were detected.

**Comment by DjSlash** [comment, marked as answer]
The default behavior of the duplicates plugin is to match **keys**: `mb_trackid` and `mb_albumid`. If those are not present, it won't match. This is probably the case here, since you imported the files without tagging them.

From the [plugin page](https://beets.readthedocs.io/en/stable/plugins/duplicates.html): Get tracks with the same title, artist, and album: `beet duplicates -k title -k albumartist -k album`.
